internal/connector/pingone/platform/resources/pingone_agreement_localization_revision.go

```diff
@@ -73,10 +73,22 @@ func (r *PingoneAgreementLocalizationRevisionResource) ExportAll() (*[]connector
 							revisionId, revisionIdOk := revision.GetIdOk()
 
 							if revisionIdOk {
+								commentData := map[string]string{
+									"Resource Type":                      r.ResourceType(),
+									"Agreement Name":                     *agreementName,
+									"Agreement Language Locale":          *agreementLanguageLocale,
+									"Revision No.":                       fmt.Sprintf("%d", (revisionIndex + 1)),
```

Comment:
Are these index values relevant for users? Would they care about the order that these come back from the API?

Reply:
Good point, I'll get these removed.